### DISCUSSION TITLE
Guide for v4?

### DISCUSSION THREAD

**Discussion by yellow1912** [maizzle/framework#634] [Q&A]
Is there any upgrade guide for v4? For some reason right now maizzle does not generate all the necessary tailwind css, so the email is kind of half styled which is weird.

**Comment by yellow1912** [comment]
I have a sample output file here:

https://gist.github.com/yellow1912/678875240f9b6903c9b688e818034229

**Comment by cossssmin** [comment]
Sorry for the lack of a proper upgrade guide, didn't have the time to finish writing it. It'll be done in a few days.

For now you can follow the setup in the [4.0 branch of the Starter](https://github.com/maizzle/maizzle/tree/4.0).

**Reply by cossssmin** [reply]
Forgot to update my reply, we do have an upgrade guide:

https://maizzle.com/docs/upgrade-guide
